<html>
<head>
<title>PyErr_BadInternalCall</title>
<meta name='robots' content='noindex,nofollow'>
<meta name='generator' content='GLOBAL-5.8.1'>
</head>
<body text='#191970' bgcolor='#f5f5dc' vlink='gray'>
<pre>
<a href='../S/3106.html#L588'>PyErr_BadInternalCall</a>  588 Python/errors.c #undef PyErr_BadInternalCall
<a href='../S/3106.html#L590'>PyErr_BadInternalCall</a>  590 Python/errors.c PyErr_BadInternalCall(void)
<a href='../S/3106.html#L595'>PyErr_BadInternalCall</a>  595 Python/errors.c #define PyErr_BadInternalCall() _PyErr_BadInternalCall(__FILE__, __LINE__)
</pre>
</body>
</html>
